en/docs/deploy-operate/secure/sso-configuration.md

```
@@ -0,0 +1,241 @@
+| End-session endpoint | Where ICP sends users to log out |
+
+### Step 2: Update `Deployment.toml`
+
+Locate `conf/Deployment.toml` under your WSO2 Integrator installation and add the following block, replacing the placeholder values with your identity provider's details.
+
+<Tabs>
+<TabItem value="macos" label="macOS" default>
+
+Default installation path: `/Applications/WSO2 Integrator.app/Contents/components/icp/conf/deployment.toml`
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)